roles/ceph-mon/defaults/main.yml

```diff
@@ -44,18 +44,24 @@ cephfs_pools:
 ###############
 # CRUSH RULES #
 ###############
-crush_rule_config: false
+# Set crush_rule_config to true (default is false) if you want ceph-ansible to configure extra crush rules for you.
+ crush_rule_config: false
```

Comment:
Please remove the extra space at the beginning of the line. Invalid syntax at playbook run.

Reply:
sorry linter issue on my side, fix